pkgs/desktops/gnome-3/core/evolution-data-server/hardcode-gsettings.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/src/libedataserver/e-source-registry.c b/src/libedataserver/e-source-registry.c
+index a5a30a3e1..0c04b62c8 100644
+--- a/src/libedataserver/e-source-registry.c
++++ b/src/libedataserver/e-source-registry.c
+@@ -116,6 +116,8 @@ struct _ESourceRegistryPrivate {
+ 	GHashTable *sources;
+ 	GMutex sources_lock;
+ 
++	GSettingsSchemaSource *schema_source;
++	GSettingsSchema *schema;
+ 	GSettings *settings;
+ 
+ 	gboolean initialized;
+@@ -1339,7 +1341,11 @@ source_registry_dispose (GObject *object)
+ 	if (priv->settings != NULL) {
+ 		g_signal_handlers_disconnect_by_data (priv->settings, object);
+ 		g_object_unref (priv->settings);
++		g_object_unref (priv->schema);
```

Comment:
Should be `g_settings_schema_source_unref` and `g_settings_schema_unref`, these are not `GObject`s.